lib/utils.js

```diff
@@ -29,31 +32,24 @@ utils.call = function (host, path, data, cb) {
       cb(null, '')
       return
     }
-
+    let body
     try {
-      var body = JSON.parse(resp.body)
+      body = JSON.parse(resp.body)
     } catch (caughtErr) {
       if (caughtErr instanceof SyntaxError) {
         // Probably a cloudflare error page
         const body = resp.body.toString()
 
-        switch (resp.statusCode) {
-          case 403:
-            if (/Request blocked\./.test(body)) {
-              err = new Error('Request blocked by CloudFlare')
-              break
-            }
-
-            if (/cf-error-code">1009/.test(body)) {
-              err = new Error('Yout IP is banned by CloudFlare')
-              break
-            }
-
-            break
-          default:
-            err = new Error('Response is not JSON. Status code: ' + resp.statusCode)
-
-            err.code = resp.statusCode
+        if (resp.statusCode === 403) {
+          if (/Request blocked\./.test(body)) {
+            err = new Error('Request blocked by CloudFlare')
+          }
+          if (/cf-error-code">1009/.test(body)) {
+            err = new Error('Your IP is banned by CloudFlare')
+          }
+        } else {
+          err = new Error('Response is not JSON. Status code: ' + resp.statusCode)
+          err.code = resp.statusCode
```

Comment:
The switch in there was to handle more 40x errors in the future but ok.

Reply:
As well as 5xx codes

Reply:
i guess it can be switched to a switch if other status codes arise.